packages/strapi/bin/strapi.js

```diff
@@ -51,7 +51,13 @@ program
 program
   .command('new')
   .option('-d, --dev', 'Development mode')
-  .description('create a new application ')
+  .option('--dbtype <dbtype>', 'Database type')
```

Comment:
What is the more easy to understand `--dbtype` or `--dbclient` ?

Reply:
I'll update to dbclient

Reply:
Done in https://github.com/strapi/strapi/pull/582/commits/1a44fd641086868db87533b7db1c32de6cf12d90